src/Servers/Kestrel/Core/src/Internal/Http/ChunkWriter.cs

```diff
@@ -5,59 +5,62 @@
 using System.Buffers;
 using System.IO.Pipelines;
 using System.Text;
+using System.Runtime.CompilerServices;
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http
 {
     internal static class ChunkWriter
     {
-        private static readonly ArraySegment<byte> _endChunkBytes = CreateAsciiByteArraySegment("\r\n");
         private static readonly byte[] _hex = Encoding.ASCII.GetBytes("0123456789abcdef");
```

Comment:
This could be changed to a `ReadonlySpan<byte>` returning property (or inlined in the method) - it would be compiled as just static data and `startHex` will be a constant then. This would get rid of the static .cctor also.